android/KMAPro/build.sh

```diff
@@ -109,3 +113,15 @@ if builder_start_action test; then
 
   builder_finish_action success test
 fi
+
+if builder_start_action publish; then
+  # Copy Release Notes
+  generateReleaseNotes
+
+  # Publish Keyman for Android to Play Store
+  echo "PUBLISH_FLAGS $PUBLISH_FLAGS"
+  cd "$KEYMAN_ROOT/android/KMAPro/"
```

Comment:
This shouldn't be necessary as the script already `cd $THIS_SCRIPT_PATH` when it starts (via builder)

```suggestion
```